pkgs/okta/okta-aws-cli/registry.yaml

```diff
@@ -0,0 +1,51 @@
+packages:
+  - type: github_release
+    repo_owner: okta
+    repo_name: okta-aws-cli
+    description: A CLI for having Okta as the IdP for AWS CLI operations
+    asset: okta-aws-cli_{{trimV .Version}}_{{.OS}}_{{.Arch}}.{{.Format}}
+    format: tar.gz
+    files:
+      - name: okta-aws-cli
+        src: okta-aws-cli_{{.Version}}
+    checksum:
+      type: github_release
+      asset: checksums.txt
+      algorithm: sha256
+    version_constraint: semver(">= 1.0.0")
+    version_overrides:
+        # checksums.txt isn't normalised
+      - version_constraint: semver("= 0.3.0")
+        checksum:
+          enabled: false
+      - version_constraint: semver(">= 0.2.0")
+        replacements:
+          amd64: x86_64
+          darwin: Darwin
+          linux: Linux
+          windows: Windows
+      - version_constraint: semver(">= 0.1.0")
+        replacements:
+          amd64: x86_64
+          darwin: Darwin
+          linux: Linux
+          windows: Windows
+        # signed.tar.gz doesn't exist in checksum file
+      - version_constraint: semver(">= 0.0.4")
+        overrides:
+          - goos: darwin
+            goarch: amd64
+            asset: okta-aws-cli_{{trimV .Version}}_{{.OS}}_{{.Arch}}_signed.{{.Format}}
+        replacements:
+          amd64: x86_64
+          darwin: Darwin
+          linux: Linux
+          windows: Windows
+        checksum:
+          enabled: false
```

Comment:
I guess we can disable only darwin/amd64.

Reply:
I think so too. Can aqua ignore only certain architectures or os??

Reply:
It should work.
https://github.com/aquaproj/aqua-registry/pull/15263/files#r1319266189